targets/linux/rpm/almalinux/v8.go

```diff
@@ -0,0 +1,30 @@
+package almalinux
+
+import (
+	"github.com/Azure/dalec/targets/linux/rpm/distro"
+)
+
+const (
+	v8TargetKey    = "almalinux8"
+	dnfCacheNameV8 = "v8-dnf-cache"
```

Comment:
To be consistent with the other `v9` file:
```suggestion
	dnfCacheNameV8 = "almalinux8-dnf-cache"
```

Reply:
Dang missed this.